docs/identity-platform/v2-oauth-ropc.md

```diff
@@ -30,6 +30,41 @@ The Microsoft identity platform supports the [OAuth 2.0 Resource Owner Password
 > * An exception to a hybrid identity federation scenario would be the following: Home Realm Discovery policy with **AllowCloudPasswordValidation** set to TRUE will enable ROPC flow to work for federated users when an on-premises password is synced to the cloud. For more information, see [Enable direct ROPC authentication of federated users for legacy applications](~/identity/enterprise-apps/home-realm-discovery-policy.md#enable-direct-ropc-authentication-of-federated-users-for-legacy-applications).
 > * Passwords with leading or trailing whitespaces are not supported by the ROPC flow.
 
+## How to migrate away from ROPC
+
+As MFA becomes more prevalent, certain Microsoft web APIs will only accept access tokens if they have passed MFA requirements. Applications and test rigs relying on ROPC will be locked out as Microsoft Entra will either refuse to issue the token or the resource will reject it.
```

Comment:
Replace "it" at the end of this sentence with the noun. Should it be replaced with something like "request"?